src/Particle/DistanceTableData.h

```diff
@@ -188,8 +188,8 @@ class DistanceTableData
    * @param P the target particle set
    */
   virtual void evaluate(ParticleSet& P) = 0;
-  virtual void mw_evaluate(const RefVectorWithLeader<DistanceTableData>& dt_list,
```

Comment:
"mw_" prefix is a signature. `mw_evaluate` corresponds to `evaluate`. So only the piece after "mw_" should follow function naming convention.

Reply:
@ye-luo  thanks, I'll revert these. Are there other special cases beside `mw_`? Asking so we can document these.

Reply:
Also "flex_" is used for upper level batched APIs. I'm not aware of others.